src/tests/EdFi.DataManagementService.Tests.E2E/Features/ResourceQueries/DescriptorQueriesQueryString.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Query String handling for GET requests for Descriptor Queries
```

Comment:
@msilesgap 
All the descriptors tests with query string are moved to this new file